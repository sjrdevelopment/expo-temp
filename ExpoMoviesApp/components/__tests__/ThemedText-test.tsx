import * as React from "react"
import renderer from "react-test-renderer"

import { ThemedText } from "../ThemedText"

it(`renders correctly`, () => {
  const tree = renderer.create(<ThemedText>Snapshot test!</ThemedText>).toJSON()

  expect(tree).toMatchSnapshot()
})

it(`renders correctly with 'title'`, () => {
  const tree = renderer
    .create(<ThemedText type="title">Snapshot test!</ThemedText>)
    .toJSON()

  expect(tree).toMatchSnapshot()
})

it(`renders correctly`, () => {
  const tree = renderer
    .create(<ThemedText type="defaultSemiBold">Snapshot test!</ThemedText>)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
it(`renders correctly`, () => {
  const tree = renderer
    .create(<ThemedText type="subtitle">Snapshot test!</ThemedText>)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
it(`renders correctly`, () => {
  const tree = renderer
    .create(<ThemedText type="link">Snapshot test!</ThemedText>)
    .toJSON()

  expect(tree).toMatchSnapshot()
})
