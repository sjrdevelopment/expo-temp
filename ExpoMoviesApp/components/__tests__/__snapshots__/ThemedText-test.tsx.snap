// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#11181C",
      },
      {
        "fontSize": 16,
        "lineHeight": 24,
      },
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
    ]
  }
>
  Snapshot test!
</Text>
`;

exports[`renders correctly 2`] = `
<Text
  style={
    [
      {
        "color": "#11181C",
      },
      undefined,
      undefined,
      {
        "fontSize": 16,
        "fontWeight": "600",
        "lineHeight": 24,
      },
      undefined,
      undefined,
      undefined,
    ]
  }
>
  Snapshot test!
</Text>
`;

exports[`renders correctly 3`] = `
<Text
  style={
    [
      {
        "color": "#11181C",
      },
      undefined,
      undefined,
      undefined,
      {
        "fontSize": 20,
        "fontWeight": "bold",
      },
      undefined,
      undefined,
    ]
  }
>
  Snapshot test!
</Text>
`;

exports[`renders correctly 4`] = `
<Text
  style={
    [
      {
        "color": "#11181C",
      },
      undefined,
      undefined,
      undefined,
      undefined,
      {
        "color": "#0a7ea4",
        "fontSize": 16,
        "lineHeight": 30,
      },
      undefined,
    ]
  }
>
  Snapshot test!
</Text>
`;

exports[`renders correctly with 'title' 1`] = `
<Text
  style={
    [
      {
        "color": "#11181C",
      },
      undefined,
      {
        "fontSize": 32,
        "fontWeight": "bold",
        "lineHeight": 32,
      },
      undefined,
      undefined,
      undefined,
      undefined,
    ]
  }
>
  Snapshot test!
</Text>
`;
